examples/medplum-patient-intake-demo/src/bots/core/intake-form.test.ts

```diff
@@ -555,6 +570,43 @@ describe('Intake form', async () => {
       expect(consents[3].scope).toStrictEqual(consentScopeMapping.adr);
       expect(consents[3].category[0]).toStrictEqual(consentCategoryMapping.acd);
       expect(consents[3].status).toStrictEqual('active');
+
+      expect(consents[4].scope).toStrictEqual(consentScopeMapping.patientPrivacy);
+      expect(consents[4].category[0]).toStrictEqual(consentCategoryMapping.cd);
+      expect(consents[4].status).toStrictEqual('active');
+      expect(consents[4].provision).toBeDefined();
+      expect(consents[4].provision?.type).toStrictEqual('permit');
+      expect(consents[4].provision?.action?.[0]).toStrictEqual(consentProvisionActionMapping.use);
+      expect(consents[4].provision?.purpose?.[0]).toStrictEqual(consentProvisionPurposeMapping.patadmin);
+      expect(consents[4].provision?.code).toStrictEqual([
+        consentProvisionCodeMapping.email,
+        consentProvisionCodeMapping.appointmentReminders,
+      ]);
+
+      expect(consents[5].scope).toStrictEqual(consentScopeMapping.patientPrivacy);
+      expect(consents[5].category[0]).toStrictEqual(consentCategoryMapping.cd);
+      expect(consents[5].status).toStrictEqual('active');
+      expect(consents[5].provision).toBeDefined();
+      expect(consents[5].provision?.type).toStrictEqual('permit');
+      expect(consents[5].provision?.action?.[0]).toStrictEqual(consentProvisionActionMapping.use);
+      expect(consents[5].provision?.purpose?.[0]).toStrictEqual(consentProvisionPurposeMapping.patadmin);
+      expect(consents[5].provision?.code).toStrictEqual([
+        consentProvisionCodeMapping.phone,
+        consentProvisionCodeMapping.sms,
+      ]);
+
+      expect(consents[6].scope).toStrictEqual(consentScopeMapping.patientPrivacy);
+      expect(consents[6].category[0]).toStrictEqual(consentCategoryMapping.cd);
+      expect(consents[6].status).toStrictEqual('active');
+      expect(consents[6].provision).toBeDefined();
+      expect(consents[6].provision?.type).toStrictEqual('permit');
+      expect(consents[6].provision?.action?.[0]).toStrictEqual(consentProvisionActionMapping.use);
+      expect(consents[6].provision?.purpose?.[0]).toStrictEqual(consentProvisionPurposeMapping.patadmin);
+      expect(consents[6].provision?.code).toStrictEqual([
+        consentProvisionCodeMapping.appointmentReminders,
+        consentProvisionCodeMapping.phone,
+        consentProvisionCodeMapping.sms,
+      ]);
```

Comment:
Looks like all three consents are set to `permit`. Perhaps switch one (test data) to `deny` so we can test both cases?